computerTeam.html

```diff
@@ -43,15 +36,16 @@
     position: relative;
 }
 .our-team .pic{
-    border-radius: 50%;
+	/* border-radius: 50%; */
+	margin-top: 10px;

```

Comment:
Try increasing height also.
It's still looking same but now in rectangular form.